### PR TITLE
global: param for serializer aliases query arg

### PR DESCRIPTION
**Pull request by ntarocco**

### CODE REVIEW

File: invenio_records_rest/views.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ def create_url_rules(endpoint, list_route=None, item_route=None,
                      delete_permission_factory_imp=None,
                      record_class=None,
                      record_serializers=None,
+                     record_serializers_query_aliases=None,
```
Comment (slint) [inline invenio_records_rest/views.py]: Maybe just `(record_)serializers_aliases`, to keep it generic? (in case somebody wants to something fancy...)

Comment (ntarocco) [reply invenio_records_rest/views.py]: done!

**Comment by ntarocco** [comment]
@nharraud see https://github.com/inveniosoftware/invenio-rest/pull/78 for to have a better understanding of the change.
The aim is to be able to call
`/api/record/<id>?format=json`
to be able to export correctly a record using a redirect without headers

File: invenio_records_rest/views.py [hunk 2]
```diff
@@ -117,6 +117,7 @@ def create_url_rules(endpoint, list_route=None, item_route=None,
                      delete_permission_factory_imp=None,
```
Comment (nharraud) [inline invenio_records_rest/views.py]: Can you fix the copyright year in this file please?

Comment (ntarocco) [reply invenio_records_rest/views.py]: Absolutely, done. Sorry I didn't notice it here.
I have also added the release commit.

**Pull request merged**